src/wiki_ulmfit.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+import gdown
+import sentencepiece as spm
+from fastai import *
+from fastai.text import *
+from gensim.corpora import WikiCorpus
+
+from src.utils import LangTokenizer
+
+logger = logging.getLogger()
+
+
+class WikiTrainer:
+    def __init__(self, path, lang="id"):
+        self.path = path
+        self.lang = lang
+
+    def download_wiki_files(self, wiki_text=False):
```

Comment:
Never mix a download function with your Trainer wheels.

Reply:
Moved download_wiki_files to download.py